decidim-core/app/helpers/decidim/layout_helper.rb

```diff
@@ -161,20 +139,34 @@ def current_user_unread_data
       end
     end
 
-    def organization_description_label
```

Comment:
@andreslucena , i believe removing this method is outside the scope of this backport. Could you revert?